### PR TITLE
Update assembly scanning information for NSB 7

### PR DESCRIPTION
**Pull request by bording**
AppDomain assemblies are now scanned by default.

### CODE REVIEW

File: nservicebus/hosting/assembly-scanning_appdomain_core_[7,].partial.md [hunk 1]
```diff
@@ -0,0 +1,6 @@
+
+## AppDomain assemblies
+
+Assemblies already loaded into the AppDomain are scanned by default. The endpoint can be configured to not scan these assemblies by using the following:
```
Comment (SimonCropp) [inline nservicebus/hosting/assembly-scanning_appdomain_core_[7,].partial.md]: `already ` is redundant?

Comment (SimonCropp) [inline nservicebus/hosting/assembly-scanning_appdomain_core_[7,].partial.md]: > The endpoint can be configured to not scan these assemblies by using the following:

how about

> AppDomain assembly scanning can be disable using:

**Comment by SimonCropp** [comment]
minor comments. but rest LGTM

**Comment by bording** [comment]
@SimonCropp Pushed up changes addressing your feedback.

**Pull request merged**